CMakeLists.txt

```diff
@@ -606,10 +605,18 @@ if(GPERF)
   find_package(Gperftools REQUIRED)
 endif()
 
-if(USE_GETTEXT)
-  find_package(Gettext)
+find_package(Gettext)
+if (Gettext_FOUND)
+  set(PO_COUNT 0)
   foreach(lang bg da de es fr hr it ko_KR pt_BR ru sv zh_CN zh_TW)
     GETTEXT_PROCESS_PO_FILES(${lang} ALL PO_FILES ${CMAKE_CURRENT_LIST_DIR}/Translations/${lang}.po)
+
+    if(ANDROID)
+      math(EXPR PO_COUNT "${PO_COUNT}+1")
+      add_custom_target(copy_${lang} ALL COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_BINARY_DIR}/${lang}.gmo" "${DevilutionX_SOURCE_DIR}/android-project/app/src/main/assets/${lang}.gmo")

```

Comment:
Here is what I think is a more correct example of how to do this from StackOverflow:

```cmake
add_custom_target(copy-test-makefile ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/input.txt)
add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/input.txt
                   COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/input.txt
                                                    ${CMAKE_CURRENT_BINARY_DIR}/input.txt
                   DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/input.txt)
```

(https://stackoverflow.com/a/55184207/181228)